app/views/admin/global_settings.html.erb

```diff
@@ -10,6 +10,13 @@
 
   <table class="table borderless" summary="Global Settings fields">
     <tr>
+      <th>Application Site URL</th>
+      <td><span class="ui-icon ui-icon-help" title='Full Site url including HTTP(S) protocol that will be used as the base callback url for tracking phishing details from visitors'></span></td>
+      <td><%= f.text_field(:site_url, placeholder: 'https://phishingfrenzy.local',
+                    required: true, class: 'form-control') %></td>
+    </tr>
+    <tr>
```

Comment:
Yes this is an additional row to add in the `SITE_URL` input field.  Back in the day when I first started coding this project I thought tables were the best way to handle this, but that was before I was leveraging the grid system and bootstrap.